torch/utils/cpp_extension.py

```diff
@@ -497,14 +495,12 @@ def CUDAExtension(name, sources, *args, **kwargs):
 
     libraries = kwargs.get('libraries', [])
     libraries.append('cudart')
-    if IS_WINDOWS:
-        libraries.append('c10')
-        libraries.append('c10_cuda')
-        libraries.append('torch_cpu')
-        libraries.append('torch_cuda')
-        libraries.append('torch')
-        libraries.append('torch_python')
-        libraries.append('_C')
+    libraries.append('c10')
+    libraries.append('c10_cuda')
+    libraries.append('torch')
```

Comment:
What is `torch` here used for? The functions should be in `torch_cpu` and `torch_cuda`, right?

Reply:
Yeah, it's supposed to be an empty library. I put `torch` in here for "good luck", in case someone ever accidentally puts some symbols in it.